File: Java/ficheros/Ejercicio2.java
package ficheros;

import java.io.BufferedReader;
import java.io.FileReader;
import java.io.IOException;

/**
 * Ejercicio 2
 * 
 * Realiza un programa que lea el fichero creado en el ejercicio anterior y que
 * muestre los números por pantalla.
 * 
 * 
 * @author Francisco Javier González Sabariego
 *
 */
public class Ejercicio2 {
  public static void main(String[] args) {
    
    String linea = "";
    
    try {
      BufferedReader r = new BufferedReader(new FileReader("ejercicio1.txt"));
      
      linea = r.readLine();
      
      while(linea != null) {
        System.out.println(linea);
        linea = r.readLine();        
      }
      
      r.close();
      
    } catch (IOException ioe) {
      System.err.println("No se ha podido leer el fichero.");
    }
    
  }
}

File: Python/ficheros/Ejercicio2.py
'''
Created on 3 abr. 2019

 * Ejercicio 2
 * 
 * Realiza un programa que lea el fichero creado en el ejercicio anterior y que
 * muestre los números por pantalla.
 * 
 * 
 * @author Francisco Javier González Sabariego

@author: FJGS_
'''

import sys

try:
    r = open('Ejercicio1.txt','r');
    
    linea = r.readline()
    
    while (True):
        print(linea, end="")
        linea = r.readline()
        if linea == '':
            break
    
    r.close()

except OSError:
    sys.stderr.write("No se ha podido leer el fichero.")